src/features/MovieItem/component.js

```diff
@@ -1,22 +1,31 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './style.scss';
 
-export default class MovieItem extends Component {
-	render() {
-		return (
-			<div className='movieItem-section'>
-				<img src={this.props.img} alt='random text' />
-				<ul className='movieItem-details'>
-					<h4>{this.props.name}</h4>
-					<h4>{this.props.date}</h4>
-				</ul>
-				<span className='movieItem-category'>{this.props.category}</span>
-				<div className='movieItem-buttons'>
-					<button className='btn btn-editItem' onClick={() => this.props.handleEditMovie(this.props.id)}>Edit</button>
-					<button className='btn btn-removeItem' onClick={() => this.props.handleRemoveMovie(this.props.id)} >Remove</button>
-				</div>
+const MovieItem = ({
+	name,
+	date,
+	category,
+	img,
+	id,
+	editMovieHandler,
+	removeMovie,
+	showMovieDetailsHandler
+}) => {
+	return(
+		<div className='movieItem-section'>
+			<img src={img} alt='random text' onClick={() => showMovieDetailsHandler(id)}/>
```

Comment:
same as in previous PR, don't use anonymous functions as props

Reply:
This one wasn't fixed

Reply:
This should be good with the latest commit. Our conversation helped me understand what was the problem